control-plane/README.md

```diff
@@ -1,2 +1,46 @@
-# Control Plane
-The control plane is used for controlling and monitoring the jobs.
+
+# Job Control plane
+
+The job control plane is used for controlling and monitoring the jobs.
+
+## Getting set up
+* Install [nodejs](https://nodejs.org/en/) + [yarn](https://yarnpkg.com/)
+* Clone the repo and run yarn install
+  * git clone https://github.com/google/privacy-sandbox-aggregation-service
+* Install [Firebase CLI](https://firebase.google.com/docs/cli#install_the_firebase_cli)
+* Update firebaseConfig variable in src/values.js
+* Run ```yarn start```
```

Comment:
Please change this to use `firebase emulators` which doesn't require to set firebase config values (can be read from the environment).

Reply:
It didn't work for me. But I used something from https://firebase.google.com/docs/hosting/reserved-urls#sdk_auto-configuration so now the user doesn't need to add their config

Reply:
I request firebase/init.json

Reply:
Awesome that's what we need :)